src/core/include/openvino/op/assign.hpp

```diff
@@ -14,7 +14,7 @@ namespace v3 {
 /// \ingroup ov_ops_cpp_api
 class OPENVINO_API Assign : public util::AssignBase {
 public:
-    OPENVINO_OP("Assign", "opset3", op::Sink, 3);
```

Comment:
Is this change related to catching Assign layer by matcher passes?

Reply:
> Is this change related to catching Assign layer by matcher passes?

Yes